### PR TITLE
[overall] fixes some bugs

### PR DESCRIPTION
**Pull request by GE-PrimeMinister**

### CODE REVIEW

File: sections/introduction.tex [hunk 1]
```diff
@@ -6,27 +6,25 @@ \section{Introduction}
 The Internet of Things(IoT) technology now penetrates various scenarios today.
 Examples include smart healthcare, industrial monitoring, and city-scale
 sensing. A classic scenario is that a set of sensors keep tracking and focus on
```
Comment (inclyc) [inline sections/introduction.tex:8]:
```suggestion
sensing. A classic scenario is that a set of sensors keep tracking and focusing on
```

File: sections/introduction.tex [hunk 2]
```diff
@@ -6,27 +6,25 @@ \section{Introduction}
...
-its metric, then upload to dedicated cloud server with timestamps. Then the
-cloud server analysis these data from multiple resources in order to generate
-knowledge like system status.
+its metric, then upload to dedicated cloud server with timestamps. The cloud
```
Comment (inclyc) [inline sections/introduction.tex:9]:
```suggestion
its metric, then upload to a dedicated cloud server with timestamps. The cloud
```

File: sections/introduction.tex [hunk 3]
```diff
@@ -6,27 +6,25 @@ \section{Introduction}
...
+server will analyze these multi-source data in order to generate knowledge like
```
Comment (inclyc) [inline sections/introduction.tex:10]:
```suggestion
server will analyze these multi-source data to generate knowledge like
```

File: sections/introduction.tex [hunk 4]
```diff
@@ -6,27 +6,25 @@ \section{Introduction}
...
 % 第二段 时序数据库的聚合、分析带来了什么挑战 -- 快速的查询聚合值
-In this context, there is a general need to store time-series data and quickly
-retrieve it for cloud server analysis, and the database provides a caching
-mechanism that enables quick queries for some common types of analysis. In
-relational database practice, queries based on aggregation operations such as
+In this context, there exists a general need to store time-series data and
+quickly retrieve it for cloud server analysis, and the database provides a
+caching mechanism that enables quick queries for some common types of analysis.
+In relational database practice, queries based on aggregation operations such as
 sum, maximum, and so on are very common, and are widely used in monitoring and
```
Comment (inclyc) [inline sections/introduction.tex:18]:
```suggestion
sum, maximum, and so on are very common and are widely used in monitoring and
```

File: sections/introduction.tex [hunk 5]
```diff
@@ -55,5 +53,5 @@ \section{Introduction}
 % 第七段 本文的结构
 In this article, we present a new data structure focus on optimization of
```
Comment (inclyc) [inline sections/introduction.tex:54]:
```suggestion
In this article, we present a new data structure focus on the optimization of
```

File: sections/introduction.tex [hunk 6]
```diff
@@ -55,5 +53,5 @@ \section{Introduction}
...
 aggregate operations. And we analysis time complexity and perform benchmark to
```
Comment (inclyc) [inline sections/introduction.tex:55]:
```suggestion
aggregate operations. And we analyzed time complexity and perform a benchmark to
```

File: sections/introduction.tex [hunk 7]
```diff
@@ -55,5 +53,5 @@ \section{Introduction}
...
-show the advance of this data structure in comparison of other popular time
+show the advanced nature of this data structure in comparison of other popular time
```
Comment (inclyc) [inline sections/introduction.tex:56]:
```suggestion
show the advanced nature of this data structure in a comparison of other popular time
```

File: sections/related_works.tex [hunk 8]
```diff
@@ -3,7 +3,7 @@ \section{Related Works}
 \subsection{Gorilla}
 
 Gorilla is an in-memory timeseries database using its dedicated compressing
-method by calculate delta of delta. Gorilla use a Timeseries map (TSmap) as its
+method by calculate ``delta of delta''. Gorilla use a Timeseries map (TSmap) as its
```
Comment (inclyc) [inline sections/related_works.tex:6]:
```suggestion
method by calculate ``delta-of-delta''. Gorilla use a Timeseries map (TSmap) as its
```

File: sections/introduction.tex (hunk 7 above)
Comment (inclyc) [reply sections/introduction.tex]: Grammerly说的(笑)

**Review by inclyc — APPROVED** [review]

File: sections/introduction.tex (hunk 7 above)
Comment (inclyc) [reply sections/introduction.tex]: 会不会改成`in comparison with`好点

Comment (GE-PrimeMinister) [reply sections/introduction.tex]: 草，刚又看了一边，搭配是`in comparison with`

Comment (GE-PrimeMinister) [reply sections/introduction.tex]: `comparsion`可做可数名词，有`a comparison of xxx`，不过这里一般`comparison`做主语

**Review by inclyc — APPROVED** [review]

File: sections/introduction.tex (hunk 7 above)
Comment (GE-PrimeMinister) [reply sections/introduction.tex]: 喵喵

**Pull request merged**